resolwe_bio/tests/processes/test_generate_report.py

```diff
@@ -0,0 +1,42 @@
+# pylint: disable=missing-docstring
+from resolwe_bio.utils.test import skipDockerFailure, BioProcessTestCase
+
+
+class ReportProcessorTestCase(BioProcessTestCase):
+
+    @skipDockerFailure("Processor requires a custom Docker image.")
```

Comment:
The issue is with one of the inputs used in the test, not with the process itself. To prepare the input correctly, we need GATK image.